python/src/equistore/operations/pow.py

```diff
@@ -45,16 +43,25 @@ def _pow_block_constant(block: TensorBlock, constant: float) -> TensorBlock:
         components=block.components,
         properties=block.properties,
     )
+    if len(block.gradients_list()) == 0:
+        return result_block
```

Comment:
I'm not sure what we gain with this early return? If there are no gradients, the loop below will be skipped anyway

Reply:
ah right we do it in the abs because before we compute `_shape` we do the `dispatch`